airbyte-config/config-models/src/main/java/io/airbyte/config/CatalogDefinitionsConfig.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config;
+
+import java.nio.file.Path;
+
+public class CatalogDefinitionsConfig {
+
+  private static final String RESOURCE_DIRECTORY_PATH = "src/main/resources/";
+  private static final String LOCAL_CONNECTOR_CATALOG_FILE_NAME = "oss_catalog.json";
+  private static final String SEED_SUBDIRECTORY = "seed/";
+
+  private static final String REMOTE_OSS_CATALOG_URL =
+      "https://storage.googleapis.com/prod-airbyte-cloud-connector-metadata-service/oss_catalog.json";
```

Comment:
Do we have our pretty url yet @cpdeethree?

Reply:
working on getting this in now! Should be ready before merge

Reply:
For reference, here's the PR adding this. You can see how I've created the other environment url's too for later use e.g. dev-1-connectors.airbyte.com

Reply:
We should continue without the fancy url, there is a problem with the tfstate on that PR, we are trying to get it resolved but it shouldn't block this url.